docs/source/extensions/logging.rst

```diff
@@ -265,7 +265,7 @@ to see fewer messages like so:
 .. code-block:: python
 
     import logging
-    logging.getLogger("lightning").setLevel(logging.ERROR)
+    logging.getLogger("pytorch_lightning").setLevel(logging.ERROR)
```

Comment:
should this not be consistent with what we have in `__init__`?
https://github.com/PyTorchLightning/pytorch-lightning/blob/ed67490d93e3f2bd46ba50cc53207492da0df73d/pytorch_lightning/__init__.py#L41

Reply:
I am heavily confused right now
All changes from this PR here #5509 
seem to have disappeared

Reply:
must be gone in the process when we released v1.2?

Reply:
ok, I am sad
Then it's better I restore the old branch and try to resend the PR
Thanks for helping me notice that @SkafteNicki